app/src/org/commcare/dalvik/activities/EntityDetailActivity.java

```diff
@@ -97,8 +97,6 @@ protected void onCreate(Bundle savedInstanceState) {
             this.mEntityContext = new Pair<Detail, TreeReference>(shortDetail, mTreeReference);
         }
 
-        Entity<TreeReference> entity = factory.getEntity(mTreeReference);

```

Comment:
random unused variable, but a somewhat expensive one to be generating unnecessarily